src/lib/theme/helpers/__snapshots__/metadata.spec.ts.snap

```diff
@@ -2,9 +2,9 @@
 
 exports[`metadata helper should compile 1`] = `
 "---
-id: xyz
-title: xyx title
-sidebar_label: xyx title
+id: 'xyz'
+title: 'xyx\\\\'s title'
```

Comment:
the output is literally `'xyx\'s title'`; these snapshots really go wild with the escaping